src/UXClient/Components/ScatterPlot/ScatterPlot.ts

```diff
@@ -374,6 +380,90 @@ class ScatterPlot extends ChartVisualizationComponent {
             .style("width", `${this.svgSelection.node().getBoundingClientRect().width + 10}px`);
     }
 
+    /******** DRAW CONNECTING LINES BETWEEN POINTS ********/
```

Comment:
@niusoff when you get a chance could you check out this function, especially the d3 update pattern.  Just want to make sure I'm getting this right : )